lib/reporters/html.js

```diff
@@ -38,7 +37,7 @@ exports = module.exports = HTML;
 
 var statsTemplate =
   '<ul id="mocha-stats">' +
-  '<li class="progress"><canvas width="40" height="40"></canvas></li>' +
+  '<li class="progress-contain"><progress class="progress-element" max="100" value="0"></progress><svg class="progress-ring"><circle class="ring-whole"/><circle class="ring-highlight" stroke-dasharray="0%,100%"/></svg><div class="progress-text">0%</div></li>' +
```

Comment:
Comparing visual changes:

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<img alt="Progress circle with 11px off-center font" src="https://github.com/mochajs/mocha/assets/3335181/cf105f0b-2abb-4c6f-a646-bfbac21d85f8" />
</td>
<td>
<img alt="Progress circle with 12px font" src="https://github.com/mochajs/mocha/assets/3335181/dca978ca-9265-46fc-8ca9-45b6fe6f51b6" />
</td>
</tr>
</tbody>
</table>

Ignoring that the text was slightly off-center before, I still see two unintended visual changes:
* The font size looks like it went from `11px` to `12px`
* The stroke thickness doubled

Let me know if you need help wrestling with the CSS to get those changes undone.

Reply:
Is this close enough? 
![difference](https://github.com/mochajs/mocha/assets/16181109/ac87844c-f301-4d0e-9774-532e28851dee)
left to right: before with canvas, after with SVG, difference, difference text removed.

The color should be the same, looking at code, but it seems to appear a bit different.

Reply:
LGTM, thanks!

Yeah the little differences are ... close enough. And I like the less pixellated circle 😄